src/main/kotlin/lotto/stretagy/RandomLottoNumberListGenerator.kt

```diff
@@ -0,0 +1,16 @@
+package lotto.stretagy
+
+import lotto.constant.MAXIMUM_NUMBER
+import lotto.constant.MINIMUM_NUMBER
+
+class RandomLottoNumberListGenerator : LottoNumberListGenerator {
+    override fun generate(): List<Int> {
```

Comment:
```suggestion
    override fun generate(): List<LottoNumber> {
```

Int가 아닌, LottoNumber를 주면 어떨까요?